tools/elf2efi.py

```diff
@@ -611,7 +611,7 @@ def elf2efi(args: argparse.Namespace):
 
     coff.Machine = pe_arch
     coff.NumberOfSections = len(sections)
-    coff.TimeDateStamp = int(os.environ.get("SOURCE_DATE_EPOCH", time.time()))
+    coff.TimeDateStamp = int(sde if (sde := os.environ.get("SOURCE_DATE_EPOCH")) else time.time())
```

Comment:
@DaanDeMeyer the use of `:=` raised the minimum python version to >= 3.8

https://github.com/systemd/systemd/blob/f477751b10178d2a5b8c6a186b03fcc04479cbd9/README#L255

Reply:
Yikes,

What's your use case where you dont have 3.9?

Reply:
> Ah, no problem on my side, it was just a notice to change the doc. But, for example on openSUSE Leap 15.6 the default is 3.6

That's insanely old actually. All the other stuff we care about supports 3.9, so I'd just bump the minimum version to 3.9 unless you care strongly about supporting 3.7

Reply:
> I'd just bump the minimum version to 3.9

For me it's ok, same minimum as `mkosi`.